Include/cpython/unicodeobject.h

```diff
@@ -70,6 +74,7 @@
 /* Check if substring matches at given offset.  The offset must be
    valid, and the substring must not be empty. */
 
+/* Py_DEPRECATED(3.3) */
 #define Py_UNICODE_MATCH(string, offset, substring) \
```

Comment:
Would it be possible to also convert Py_UNICODE_MATCH and PyUnicode_WSTR_LENGTH to static inline functions? Maybe a second macro is needed to accept any type: see Py_INCREF() for example.

Reply:
I think it's safe to just remove this.
Note that both of `(substring)->wstr` and `(substring)->wstr_length` are used.

* If substring is PyASCIIObject*, `->wstr_length` is error.
* If substring is PyCompactUnicodeObject*, `->wstr` is error.  (`->_base.wstr` should be used).

I can not imagine who use this macro without compile error.